testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/WaitUtils.java

```diff
@@ -149,31 +148,11 @@ public static void waitForPageToLoad() {
             }
         }
 
-        WebDriverWait wait = new WebDriverWait(getCurrentDriver(), PAGELOAD_TIMEOUT_MILLIS / 1000);
-        ExpectedCondition waitCondition = null;
-
-        // Different wait strategies for Admin and Account Consoles
-        if (currentUrl.matches("^[^\\/]+:\\/\\/[^\\/]+\\/auth\\/admin\\/.*$")) { // Admin Console
```

Comment:
Considering this regex was matching `/auth` (which has been gone since a while), I believe this code path was probably dead since a while.